addons/pos_restaurant/static/src/js/Screens/FloorScreen/FloorScreen.js

```diff
@@ -77,6 +78,33 @@ odoo.define('pos_restaurant.FloorScreen', function (require) {
                 .get_table_orders(table)
                 .filter(o => o.orderlines.length !== 0 || o.paymentlines.length !== 0).length;
         }
+        _onPinchMove(ev) {
+            const touches = ev.touches;
+            // If two pointers are down, check for pinch gestures
+            if (touches.length === 2) {
+                // Calculate the current distance between the 2 fingers
+                const deltaX = touches[0].pageX - touches[1].pageX;
+                const deltaY = touches[0].pageY - touches[1].pageY;
+                const curDiff = Math.hypot(deltaX, deltaY);
+                if (this.state.pinchLastDeltaDiff === null) {
```

Comment:
the scale may be weird as `this.state.pinchLastDeltaDiff` is set only once at the first `_onPinchMove()`